uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Document.php

```diff
@@ -11,7 +11,7 @@ class CRM_Tasksassignments_BAO_Document extends CRM_Tasksassignments_DAO_Documen
    * Create a new Document based on array-data
    *
    * @param array $params key-value pairs
-   * @return CRM_Tasksassignments_DAO_Document|NULL
```

Comment:
Why did you need to add the object type? Note that we already say this method returns a Document DAO instance